### ISSUE TITLE
Exported names conflicting within Interact macros

### ISSUE THREAD

**Issue by juliohm** [JuliaGizmos/Interact.jl#190]
Minimum example:

```julia
using GeoStats
using Interact

@manipulate for a in linspace(0,1)
end

WARNING: both Reactive and GeoStats export "value"; uses of it in module Main must be qualified
UndefVarError: value not defined

Stacktrace:
 [1] display_dict(::Reactive.Signal{Any}) at /home/juliohm/.julia/v0.6/Interact/src/IJulia/setup_old.jl:96
 [2] execute_request(::ZMQ.Socket, ::IJulia.Msg) at /home/juliohm/.julia/v0.6/IJulia/src/execute_request.jl:182
 [3] eventloop(::ZMQ.Socket) at /home/juliohm/.julia/v0.6/IJulia/src/eventloop.jl:8
 [4] (::IJulia.##14#17)() at ./task.jl:335
```

As the message says, both packages export `value`. I wonder if the Interact macros could call these names exported by `Reactive.jl` explicitly to avoid conflicts?

**Comment by JobJob** [comment]
Yeah, there were some recent changes in this direction on master Interact. Maybe checking out master fixes it for you?

**Comment by JobJob** [comment]
> Maybe checking out master fixes it for you?

Had a quick look and don't think it would actually. 

Be happy if you wanted to make a PR for this - I think you'd just have to modify the 3 or so calls to `value(...)` in https://github.com/JuliaGizmos/Interact.jl/blob/master/src/IJulia/setup_old.jl and https://github.com/JuliaGizmos/Interact.jl/blob/master/src/IJulia/setup.jl

**Comment by JobJob** [comment]
Should be fixed on master thanks to #188, let us know if not.
